src/scene/scene.js

```diff
@@ -147,6 +150,10 @@ class Scene extends EventHandler {
         this._lightmapFilterRange = 10;
         this._lightmapFilterSmoothness = 0.2;
 
+        // clustered lighting
+        this.worldClustersParams = new WorldClustersParams();
```

Comment:
Not `clusteredLightingParams`?

Reply:
We have `WorldClusters` and `WorldClustersParams' classes. I usually keep the name of the instance of the classes matching, especially if those are private members.